documentation/Makefile

```diff
@@ -16,9 +16,13 @@ libcom_HEADERS += alarm
 libcom_HEADERS += alarmString
 libcom_HEADERS += asTrapWrite
 libcom_HEADERS += bucketLib
+libcom_HEADERS += cadef
+libcom_HEADERS += caerr
+libcom_HEADERS += caeventmask
```

Comment:
Channel Access is not part of libCom. All these CA header filenames should be added to a new `ca_HEADERS` variable, add `ca` to `HEADER_TYPES`, and create a new file ca-API.rst